src/tigerbeetlex.zig

```diff
@@ -7,26 +7,26 @@ const account_batch = @import("account_batch.zig");
 const client = @import("client.zig");
 const id_batch = @import("id_batch.zig");
 const transfer_batch = @import("transfer_batch.zig");
+const account_filter_batch = @import("account_filter_batch.zig");
 const AccountBatch = account_batch.AccountBatch;
 const IdBatch = id_batch.IdBatch;
 const TransferBatch = transfer_batch.TransferBatch;
+const AccountFilterBatch = account_filter_batch.TransferBatch;
 const Client = client.Client;
 
 const ClientResource = client.ClientResource;
 const AccountBatchResource = account_batch.AccountBatchResource;
 const IdBatchResource = id_batch.IdBatchResource;
 const TransferBatchResource = transfer_batch.TransferBatchResource;
+const AccountFilterBatchResource = account_filter_batch.AccountFilterBatchResource;
 
 pub const vsr_options = .{
-    .config_base = .default,
-    .config_log_level = std.log.Level.info,
-    .tracer_backend = .none,
+    .config_verify = true,
     .hash_log_mode = .none,
     .git_commit = null,
     // TODO: take these from a proper place
-    .release = "0.15.4",
-    .release_client_min = "0.15.4",
-    .config_aof_record = false,
+    .release = "0.16.18",
+    .release_client_min = "0.16.18",
```

Comment:
`release_client_min` can be left as-is (TigerBeetle keeps it at `0.15.3` by default)

Reply:
```suggestion
    .release_client_min = "0.15.4",
```